### ISSUE TITLE
How to connect via Video?

### ISSUE THREAD

**Issue by callsanjeevat** [devin-la/ionic-video-chat-v2-issues#83]
Is there User and code documentation somewhere? I installed the server and the client. I am using ionic serve to test. I can chat between 2 users but cannot find the option to do a video call.

**Comment by oriongunning** [comment]
Yes the interface is very confusing... on the contact list, long click on someone will bring up an popup
